deploy/example/snapshotclass-nfs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshotClass
+metadata:
+  name: csi-nfs-snapclass
+driver: nfs.csi.k8s.io
+deletionPolicy: Delete
+parameters:
+  server: nfs-server.default.svc.cluster.local
```

Comment:
I don't think we need to expose these parameters, if a user wants to take a snapshot, these parameters are already defined in original storage class.